src/tokenizer.rs

```diff
@@ -39,6 +39,7 @@ symbols!(
 
     // TODO: This only is valid in Roblox
     FatArrow => "=>",
+    ThinArrow => "->",
```

Comment:
This leaves `FatArrow` unused, can you remove that?

Reply:
Pushed all requested changes.